docs: Added release notes for #5493 (#5556)
dashpay/dash: doc/release-notes-5493.md

> Masternode Reward Changes
> ------------------------

Reviewer: Nit
```suggestion
Masternode Reward Changes
-------------------------
```

> Since v19, EvoNodes are getting paid 4 block in a row.
> Once Masternode Reward Location Reallocation activates, EvoNodes will be paid as usual: one block in a row. 

Reviewer: Maybe expand to something like this?
```suggestion
Since v19, EvoNodes are paid 4 blocks in a row.
Once Masternode Reward Location Reallocation activates, all masternodes, including EvoNodes, will be paid as usual by Core in a single block. The remaining rewards for EvoNodes will be paid out by Platform.
```

> Once Masternode Reward Location Reallocation activates, all masternodes, including EvoNodes, will be paid as usual by Core in a single block. The remaining rewards for EvoNodes will be paid out by Platform.

Reviewer 2: New line
```suggestion
Once Masternode Reward Location Reallocation activates, all masternodes, including EvoNodes, will be paid as usual by Core in a single block. The remaining rewards for EvoNodes will be paid out by Platform.

```
